source/browseMode.py

```diff
@@ -416,10 +417,20 @@ def _quickNavScript(self,gesture, itemType, direction, errorMessage, readUnit):
 		item.moveTo()
 
 	@classmethod
-	def addQuickNav(cls, itemType, key, nextDoc, nextError, prevDoc, prevError, readUnit=None):
+	def addQuickNav(
+			cls,
+			itemType: str,
+			key: Optional[str],
+			nextDoc: str,
+			nextError: str,
+			prevDoc: str,
+			prevError: str,
+			readUnit: Optional[str] = None
+	):
 		"""Adds a script for the given quick nav item.
-		@param itemType: The type of item, I.E. "heading" "Link" ...

```

Comment:
Why was this line removed?

Reply:
Ugh, that must have been an accident. Restoring now.